app/controllers/application_controller.rb

```diff
@@ -8,4 +8,5 @@ class ApplicationController < ActionController::Base
   #     redirect_to new_user_session_path, alert: 'error!! change this error to flash later'
   #   end
   # end
+
```

Comment:
Look at the code above. Its all commented out. Do you need it? If not remove it.